src/components/SidebarContent.js

```diff
@@ -1,55 +1,42 @@
 import React from "react";
-import { Link } from "react-router-dom";
 import { storeCustomers } from "../data";
 import { CustomerConsumer } from "./CustomerContext";
 
 export default function SidebarContent() {
-  const updateCustomer = (ctx) => {
-    return (e) => {
-      ctx.setCustomer(e.target.value);
-    };
-  };
-
   return (
     <div className="side-menu-container">
       <CustomerConsumer>
         {(ctx) => {
-          console.log(ctx.customer);
           return (
-            <select
-              id="storeCustomers"
-              onChange={updateCustomer(ctx)}
-              defaultValue={"DEFAULT"}
-            >
-              <option value="DEFAULT" disabled>
-                Select customer
-              </option>
+            <>
+              <select
+                id="storeCustomers"
+                onChange={(e) => ctx.setCustomer(e.target.value)}
```

Comment:
since you simplified that `updateCustomer` function, I just went one step further and inlined it. Even simpler.